lightning-persister/src/lib.rs

```diff
@@ -330,3 +333,15 @@ mod tests {
 		added_monitors.clear();
 	}
 }
+
+#[cfg(all(test, feature = "unstable"))]
+pub mod bench {
+	use test::Bencher;
+
+	#[bench]
+	fn bench_sends(bench: &mut Bencher) {
```

Comment:
I think we need to add this to CI to run?

Reply:
There's already a CI run for benchmarks (because they rely on rust nightly/unstable APIs). Sadly the router benchmarks currently fail, but that is fixed by #815.

Reply:
Ah okay, I only see `test ln::channelmanager::bench::bench_sends` in CI output, not the `persister` version, but I think I'm missing something

Reply:
It shows up when I run it locally. In CI the router benchmarks fail before we get there, so it won't run yet :(